docs/content_block_manager/adr/0002-use-publishing-api.md

```diff
@@ -0,0 +1,122 @@
+potentially open up the Publishing API to consumers outside of Publisher apps. [Skip to ‘Decision’](#decision) to 
+see more detail, or read on to see the options we considered.**
+
+## Options considered
+
+### Create a standalone Content Block API
+
+Currently the CBM repo sits within an engine within Whitehall. Is it time for the app to become its own repo, with
+its own database as a source of truth?
+
+Pros:
+* We could start building it with only the needs of CBM in mind, without treading on Publishing API or Whitehall code.
+* We’d have full control over endpoint authentication/authorization and the shape of the API contract
+* We could build it as consumer-agnostic as possible from the beginning
+* Clear code ownership and team responsibility
+
+Cons:
+* Another microservice! with all its potential drawbacks of maintainability, might be left to degrade if our team
+changes or the department’s priorities shift.
+* It would not fit with the wider vision for government architecture - we don’t need yet another source of truth for
+content
+
+### Directly call either Content Store or Search API
+
+According to the lore of the Publishing ecosystem - either the Content Store or the Search API should be the
+front-facing app for our content. These are queried by the frontend and contain the definitive content for our 
+citizen users. There’s a few issues with this approach for Content Blocks:
+* The Content Store is designed to return one Document at a base path, it’s not quite the API we might need to 
+  return  e.g. lists of Blocks of a certain type, or containing a certain word.
+* Content Blocks aren’t public-facing. They don’t currently have base paths so they are not published to the Content 
+  Store, because they aren’t designed to be treated like published Editions - they could have metadata on them like 
+  Instructions to Publishers that we might not want seen out of context in the public eye. They also aren’t designed 
+  to exist on their own without the context of being embedded in a Host Document.
+
+### Use Publishing API as the source of truth
+
+We could use the existing endpoints on Publishing API, e.g. `get_content`, to get our published Content Blocks from
+outside of the publishing ecosystem.
+
+Pros:
+* Publishing API is already the source of truth, no extra backend work needed as we can keep existing publishing 
+  pipelines.
+* We’d use existing endpoints initially e.g. calling `Services.publishing_api.get_content`
+* This potentially aligns with a future GraphQL-powered use of the Publishing API as a frontend-facing app.
+
+Cons:
+* If we did open up the Publishing API to be queried externally, we'd have to be very careful about how we go about 
+  changing this. Although the precedent and the vision for this has already been started by the GraphQL work, which has opened up an endpoint to be queried directly by frontend.
+* We’d potentially need custom code in each service to call the API from backend. 
```

Comment:
Might be worth mentioning _why_ we'd have to do this - namely because there's no public HTTP endpoint, so we can't call Publishing API client side

Reply:
good call, done!